app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardMobileActivity.kt

```diff
@@ -320,7 +326,7 @@ class TeiDashboardMobileActivity :
                     if (pageType == DashboardPageType.TEI_DETAIL && programUid != null) {
                         binding.toolbarTitle.visibility = View.GONE
                         binding.editButton.visibility = View.VISIBLE
-                        binding.syncButton.visibility = View.VISIBLE
```

Comment:
syncButton should not be removed directly if visibility is always gone?

Reply:
Thanks, this was an error, it should be visible when user is on analytics or relationships.